www/index.php

```diff
@@ -377,6 +377,15 @@
                                         <input id="time" type="text" class="text short" name="time" value=""> seconds
                                     </li>
                                     <li>
+									<li>
+                                        <label for="time">

```

Comment:
You're right. I wrote it originally to prevent the user from overriding the Time Limit set in the wptdriver.ini file, but since I changed the code this is no longer necessary